sphinx/types.rst

```diff
@@ -1,104 +1,170 @@
+ time                 datetime64[ns]                       datetime64[ns]
+ power                float                                kW
+ energy               float                                kWh
+==================== ==================================   =============================
+
+Tower
+=====
+
+:py:attr:`PlantData.tower` is configured by the :py:class:`openoa.plant.TowerMetaData` class, which is set in the configuration
+data with the "tower" key. Users can set each of the following "Field Name" keys with their own
+data's column names in their met tower data, plus the "freq" field. Users just have to ensure that
+the columns are already using the specified units, and that each column is already using the listed
+data type or can be converted to that type.
+
+==================== ==================================   =============================
+ Field Name           Data Type (TowerMetaData.dtypes)     Units (TowerMetaData.units)
+==================== ==================================   =============================
+ time                 datetime64[ns]                       datetime64[ns]
+ id                   string                               None
+==================== ==================================   =============================
+
+Curtail
+=======
+
+:py:attr:`PlantData.curtail` is configured by the :py:class:`openoa.plant.CurtailMetaData` class, which is set in the configuration
+data with the "curtail" key. Users can set each of the following "Field Name" keys with their own
+data's column names in their curtailment data, plus the "freq" field. Users just have to ensure that
+the columns are already using the specified units, and that each column is already using the listed
+data type or can be converted to that type.
+
+==================== ====================================   ===============================
+ Field Name           Data Type (CurtailMetaData.dtypes)     Units (CurtailMetaData.units)
+==================== ====================================   ===============================
+ time                 datetime64[ns]                         datetime64[ns]
+ curtailment          float                                  percent
+ availability         float                                  percent
+ net_energy           float                                  kW
+==================== ====================================   ===============================
+
+Status
+======
+
+:py:attr:`PlantData.status` is configured by the :py:class:`openoa.plant.StatusMetaData` class, which is set in the configuration
+data with the "status" key. Users can set each of the following "Field Name" keys with their own
+data's column names in their turbine status data, plus the "freq" field. Users just have to ensure
+that the columns are already using the specified units, and that each column is already using the
+listed data type or can be converted to that type.
```

Comment:
I suggest mentioning that OpenOA currently doesn’t use Status data for any analyses, so this is more of a placeholder.

Reply:
I've added a note, but let me know if you'd like more/less, or something different here.